core/sail/extensible-store/src/test/java/org/eclipse/rdf4j/sail/extensiblestoreimpl/TransactionIsolationAndWalTests.java

```diff
@@ -71,6 +70,8 @@ public void testReadCommittedLargeTransaction() throws InterruptedException {
 
 		assertFalse(failure.get());
 
+		repository.shutDown();
```

Comment:
On line 45 you are printing to stdout - could you replace that with using the logger, probably to output to logger.warn level?